client/cl_main.lua

```diff
@@ -0,0 +1,322 @@
+local jobs = exports.qbx_core:GetJobs()
+local gangs = exports.qbx_core:GetGangs()
+local DynamicMenuItems = {}
+
+function CommaValue(amount)
+    local numChanged
+
+    repeat
+        amount, numChanged = string.gsub(amount, '^(-?%d+)(%d%d%d)', '%1,%2')
+    until numChanged == 0
+
+    return amount
+end
+
+---@return table
+function FindPlayers()
+	local playerCoords = GetEntityCoords(cache.ped)
+    local closePlayers = lib.getNearbyPlayers(playerCoords, 10, false)
+    for _, v in pairs(closePlayers) do
+        v.id = GetPlayerServerId(v.id)
+    end
+	return lib.callback.await('qbx_management:server:getplayers', false, closePlayers)
+end
```

Comment:
```suggestion
local function findPlayers()
    local closePlayers = lib.getNearbyPlayers(GetEntityCoords(cache.ped), 10)
    for _, v in pairs(closePlayers) do
        v.id = GetPlayerServerId(v.id)
    end
    return lib.callback.await('qbx_management:server:getplayers', false, closePlayers)
end
```

I would refactor this function slightly, as it's only used in this file use local variable, if something is not used more than a single time like playerCoords was just put it inside whereever it's needed.

Reply:
Did this one mostly for readability but it's still pretty clear what is going on. Changed